### DISCUSSION TITLE
Set up the option.txt file in modrinth, which will then be shared between all instances.

### DISCUSSION THREAD

**Discussion by Pierrelncls** [modrinth/.github#144] [Feature requests]
This would save the headache of redoing keys and parameters for each instance.

**Comment by omerien** [comment]
the options.txt file has changed format between versions, if you share it all like in the main Minecraft Launcher it causes issues (try setting up your keys in 1.8 and then use the same options.txt in 1.20 you'll get problems)

**Reply by JustForFunFoss** [reply]
What of there are multiple setups for different versions (eg one options set for pre [insert format change versian] and one for post [insert format change versian].  I generally only use 1.19, and 1.20 instances,  sosetting up only one or two key bind sets is ok, instead of all the modded instances i try.

**Comment by junaga** [comment]
This won't work, a simple symlink is not compatible with modpacks. for this feature to work an entire Minecraft mod that interops a lot of different projects from x-unknown users would be required.

**Comment by Pierrelncls** [comment]
Or you could have a setting in Modrinth that only changes the basic controls of vanilla Minecraft, while for mods you’d still need to configure them manually. I think it’s possible for Modrinth to edit the options.txt file when an instance is created to adjust these basic controls.
